app/styles/base.css

```diff
@@ -260,6 +260,10 @@ select:active {
  * ----------------------------------------
  */
 
+.noVNC_key_verification_layer {
+  z-index: 55;
+}
```

Comment:
Sorry that I'm not an expert of frontend developing. When I tried to put them in the same layer, neither of them appeas in the center of the screen. I guess it's because the elements are hidden by `visibility: hidden` not `display: none` so I just put them in the different layers and it seems to work. Is there a better solution for that issue?

Reply:
Let me have a look and see what needs to be done.

Reply:
Ah, I see the problem. If you put them in the same `<div>` then the centering is broken, yes. But keep it in a separate `<div>`, just like you have now. Then just change the class from `noVNC_key_verification_layer` to `noVNC_connect_layer`.